include/fastdds/dds/topic/TopicDataType.hpp

```diff
@@ -64,27 +64,13 @@ class TopicDataType
     /**
      * @brief Constructor
      */
-    FASTDDS_EXPORTED_API TopicDataType();
+    FASTDDS_EXPORTED_API TopicDataType() = default;
 
     /**
      * @brief Destructor
      */
     FASTDDS_EXPORTED_API virtual ~TopicDataType();
```

Comment:
```suggestion
    FASTDDS_EXPORTED_API virtual ~TopicDataType() = default;
```

Reply:
This would allow removing the corresponding source file.

Reply:
Done in 3984949b2